.evergreen/config.yml

```diff
@@ -891,27 +890,13 @@ tasks:
         vars:
           targets: check-fmt
 
-  - name: sa-errcheck
-    tags: ["static-analysis"]
-    commands:
-      - func: run-make
-        vars:
-          targets: errcheck
-
   - name: sa-lint
     tags: ["static-analysis"]
     commands:
       - func: run-make
         vars:
           targets: lint
 
-  - name: sa-vet
```

Comment:
To clarify, `sa-vet` and `sa-errcheck` are now replaced by the single `sa-lint` that is now `golangci-lint` instead of `golint`?

Reply:
Nice, I like that these are consolidated under `sa-lint`.